src/tests/ftest/util/command_utils_base.py

```diff
@@ -245,6 +256,30 @@ def update(self, value, name=None, append=False):
         self.log.debug("  Added the directory: %s => %s", name, self.value)
 
 
+class KeywordParameter(BasicParameter):
+    """A class for parameters whose values are read from a yaml file.
+
+    Supports mapping regular expression keywords to a function of the raw value.
+    """
+
+    KEYWORD_DICT = {
+        "^python://": (lambda s: literal_eval(re.sub(r'^python://(.*)$', r'str(\1)', s)))
+    }
+
+    @BasicParameter.value.getter
+    def value(self):
```

Comment:
(pylint-invalid-overridden-method) Method 'value' was expected to be 'property', found it instead as 'method'